nemo/utils/dllogger.py

```diff
@@ -15,10 +15,10 @@
 from argparse import Namespace
 from pathlib import Path
 
+from lightning_fabric.utilities.logger import _convert_params, _flatten_dict, _sanitize_callable_params
```

Comment:
a solution to removing this high level import is to only call it within the log call. This will lazily instantiate these functions and also require `dllogger` to have a Lightning version of greater than 1.9.0.